pallets/runtime/develop/src/runtime.rs

```diff
@@ -374,6 +374,8 @@ parameter_types! {
     pub const MaxValidatorPerIdentity: Permill = Permill::from_percent(33);
     // 0.05%. The higher the value, the more strict solution acceptance becomes.
     pub MinSolutionScoreBump: Perbill = Perbill::from_rational_approximation(5u32, 10_000);
+    pub const MaxInflatedTotalIssuance: Balance = 1_000_000_000 * POLY;
+    pub NonInflatedTotalYearlyReward: Balance = Perbill::from_percent(5) * MaxInflatedTotalIssuance::get();
```

Comment:
Let's use a constant instead of %. They mean the same thing but a constant amount is clearer to understand. Also, let's set the default amount to 200m (20% of 1b).

Reply:
I don't think understanding the % needs rocket science. But I think it is more of a question for the business guys how they want to play as in percentage or what amount of tokens or % of tokens they want to cap. Anyway, I will change the nos. but those are subjected to change as well.